docs/aggregation.md

```diff
@@ -64,12 +64,12 @@ Console.WriteLine(response.RawResponse);
 ```
 **Sample Output:**
 
-| PvEstimate | Percentage | PeriodEnd | Period |
+| Percentage | PvEstimate | PeriodEnd | Period |
```

Comment:
@nik-solcast - this has "changed" because it returns the parameters in the order specified in the arguments.

https://api.solcast.com.au/data/forecast/aggregations?collection_id=tw_citycounty_total&aggregation_id=64_kaohsiung_city&output_parameters=pv_estimate,percentage&format=csv

https://api.solcast.com.au/data/forecast/aggregations?collection_id=tw_citycounty_total&aggregation_id=64_kaohsiung_city&output_parameters=percentage,pv_estimate&format=csv

I think this is desirable behavior as it helps when new parameters are introduced

Reply:
Make sense. In this case should have similar behavior at least for live as well, if not generally.

https://api.solcast.com.au/data/live/aggregations?collection_id=tw_citycounty_total&aggregation_id=64_kaohsiung_city&output_parameters=pv_estimate,percentage&format=csv

https://api.solcast.com.au/data/live/aggregations?collection_id=tw_citycounty_total&aggregation_id=64_kaohsiung_city&output_parameters=percentage,pv_estimate&format=csv

Reply:
Yeah, live should do the same (but it uses the old formatter pathway). I'll update in TECH-488.